docs/tables/aws_codedeploy_deployment_group.md

```diff
@@ -0,0 +1,105 @@
+# Table: aws_codedeploy_app
+
+A deployment group is the AWS CodeDeploy entity for grouping EC2 instances or AWS Lambda functions in a CodeDeploy deployment. For EC2 deployments, it is a set of instances associated with an application that you target for a deployment.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  arn,
+  deployment_group_id,
+  deployment_group_name
+  application_name,
+  deployment_style,
+  region
+from
+  aws_codedeploy_deployment_group
+where
+  application_name = 'abc';
```

Comment:
Now since `application_name` is not required KeyColumn, its not required to be in where clause in example queries